docs/static/ls-ls-config.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[ls-to-ls]]
+=== Configuring Logstash-to-Logstash Communication
+
+In a complex network environment, you may need two Logstash machines talking to each other. That is, the output from the first Logstash machine becomes the input for the second Logstash machine. You may need this configuration if the data path crosses network or firewall boundaries, for example.
+
+NOTE: Don't complicate your configuration unnecessarily. If you don't have a compelling need for Logstash-to-Logstash communication, then don't implement it.
+
+[[ls-to-ls-overview]]
+==== Configuration overview
+
+. Use the Lumberjack output plugin on the first Logstash machine.
+. Use the Beats input plugin on the second Logstash machine.
+. Generate a trusted SSL certificate (required by the lumberjack protocol).
+. Save the SSL certificate on the first Logstash machine.
+. Save the SSL certificate and key on the second Logstash machine.
+. Test it.
+
+[[generate-self-signed-cert]]
+===== Generate a self-signed SSL certificate and key
+
+You can use the `openssl req` command with some specific flags to generate a self-signed certificate and key. The `openssl req` command is available on any operating system that comes with SSL.
+
+Run the following command:
+
+[source,shell]
+----
+openssl req -x509 -batch -nodes -newkey rsa:2048 -keyout lumberjack.key -out lumberjack.crt -subj /CN=localhost
+----
+
+where:
+
+* `lumberjack.key` specifies the name of the SSL key to be created
+* `lumberjack.cert` specifies the name of the SSL certificate to be created
+* `localhost` specifies the name of the first Logstash computer
+
+
+This command produces output similar to the following:
+
+[source,shell]
+----
+Generating a 2048 bit RSA private key
+.................................+++
+....................+++
+writing new private key to 'lumberjack.key'
+----
+
+[[save-cert-ls1]]
+===== Save the SSL certificate on the first Logstash machine
+
+On the first Logstash machine, run the following command:
+[source,shell]
+----
+bin/logstash --path.data /tmp/z -e 'input { generator { count => 5 } } output { lumberjack { codec => json hosts => "localhost" ssl_certificate => "lumberjack.crt" port => 5000 } }'
```

Comment:
We should explain why we're setting `path.data` here, which is to not have two logstashes attempt to use the same data path, which would prevent one from starting. A note saying that we're only doing this to be able to quickly demo this makes sense.

Reply:
Hmmmm... I meant to modify that sample based on the premise that this was a symptom of the demo and not likely in real world set up. In other words, I'm trying to help people get set up rather than helping them demo the solution.  1. Is that an accurate assumption? 2. Is setup rather than demo the correct approach for the reference guide?